app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -683,11 +685,11 @@ $include-html-global-classes: $include-html-classes;
 
 // We use this to style the glowing effect of inputs when focused
 // $glowing-effect-fade-time: 0.45s;
-// $glowing-effect-color: $input-focus-border-color;
+$glowing-effect-color: $input-focus-border-color;
 
 // Select variables
-// $select-bg-color: $ghost;
-// $select-hover-bg-color: scale-color($select-bg-color, $lightness: -3%);
+$select-bg-color: rgba( $black, .2);
```

Comment:
`.2` should be written with a leading zero as `0.2`<br>Expected 0 spaces between parentheses instead of 1